src/mono/nuget/Directory.Build.targets

```diff
@@ -1,5 +1,13 @@
 <Project>
   <UsingTask TaskName="GenerateFileFromTemplate" AssemblyFile="$(WorkloadTasksAssemblyPath)" />
   <Import Project="..\Directory.Build.targets" />
+  
+  <PropertyGroup>
+    <!-- Central place to set the versions of all mono pkgprojs. -->
+    <PackageVersion Condition="'$(PackageVersion)' == ''">$(ProductVersion)</PackageVersion>
+    <StableVersion Condition="'$(StabilizePackageVersion)' == 'true' and '$(StableVersion)' == ''">$(PackageVersion)</StableVersion>
```

Comment:
the coreclr version of this also checks for `'$(PreReleaseVersionLabel)' != 'servicing'` do we need this here too?

https://github.com/dotnet/runtime/blob/cb52e3b88177cb86929b829d92b03561ad4e806b/src/coreclr/.nuget/Directory.Build.targets#L7

Reply:
No, as the coreclr packages intentionally don't ship during servicing which isn't true for the mono packages.